PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -97,6 +100,18 @@ function TouchJump:Create(parentFrame)
 		end
 	end)
 	
+	if humanoid then
+		
+		humanoid.Changed:connect(function(prop)
```

Comment:
Oh yeah, I completely forgot PlayerScripts don't reset on respawn. As it is now, when you respawn, humanoid will still be pointing to the old humanoid.